ospf6d/ospf6_lsdb.h

```diff
@@ -25,10 +25,25 @@
 #include "table.h"
 #include "ospf6_route.h"
 
+/* LSA Statistics */
+struct ospf6_lsdb_stats {
+	uint32_t num_type1; /* Router-LSA Count */
+	uint32_t num_type2; /* Network-LSA Count */
+	uint32_t num_type3; /* Inter-Area-Prefix-LSA Count */
+	uint32_t num_type4; /* Inter-Area-Router-LSA Count */
+	uint32_t num_type5; /* AS-External-LSA Count */
+	uint32_t num_type7; /* NSSA-LSA Count */
+	uint32_t num_type8; /* Link-LSA Count */
+	uint32_t num_type9; /* Intra-Area-Prefix-LSA Count */
```

Comment:
As much as I like how this looks, please put the comments above the names on their own lines. Style tooling doesn't like it.

Reply:
fixed.